src/components/withdrawals/CreateForm.tsx

```diff
@@ -42,8 +43,7 @@ export default function CreateForm() {
   const { data: campaigns } = useCampaignList()
   const { data: personList } = usePersonList()
   const { data: vaults } = useVaultsList()
-  const currencies = Object.keys(Currencies)
-  const [currency, setCurrency] = useState('')
+
   const [approvedById, setApprovedById] = useState('')
```

Comment:
@dpmarkov We can do one more issue for t he `approvedById` replacement with `<PersonSelect />`

Reply:
I'm on it.